docs/apiSpecifications/Receipt API.yml

```diff
@@ -587,20 +591,13 @@ components:
           type: array
           items:
             type: integer
+            minimum: 0

```

Comment:
for integers we could use `minimum: 1` instead of
`minimum: 0
exclusiveMinimum: true` as it is simpler and it means the same thing